table/table_properties.cc

```diff
@@ -213,6 +214,30 @@ TableProperties::GetAggregatablePropertiesAsMap() const {
   return rv;
 }
 
+std::size_t TableProperties::ApproximateMemoryUsage() const {
+  std::size_t usage = 0;
+#ifdef ROCKSDB_MALLOC_USABLE_SIZE
+  usage += malloc_usable_size((void*)this);
+#else
+  usage += sizeof(*this);
+#endif  // ROCKSDB_MALLOC_USABLE_SIZE
+
+  std::pair<const std::string*, const std::string*> string_prop_start_end_pos =
+      TEST_GetStringPropStartEndPosition(this);
```

Comment:
The function is called TEST_* but it doesn't appear to be called only in test code.

Reply:
Done by renaming as  Get*PropStartEndPosition